hudi-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadata.java

```diff
@@ -260,6 +265,20 @@ public synchronized void close() throws Exception {
     logRecordScanner = null;
   }
 
+  /**
+   * Return the timestamp of the latest synced instant.
+   */
+  @Override
+  public Option<String> getSyncedInstantTime() {
```

Comment:
seems like same code as in the writer? any chance for code reuse?

Reply:
I dont see this on the writer side. Writer side uses metadata.getSyncedInstantTime()